AuxTel/AuxTel-Troubleshooting/General-Troubleshooting/Images-out-of-focus.rst

```diff
@@ -0,0 +1,128 @@
+.. This is a template for Image out of focus when some part of the observatory enters an abnormal state. This comment may be deleted when the template is copied to the destination.
+
+.. Review the README in this procedure's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Carlos Morales*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Ioana Sotuela, Karla Aubel*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Image-out-of-focus-Procedure:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+Image out of focus
+#########################
+
+.. note::
+    This is a procedure template file that is associated with a template directory structure. This note should be deleted when the section is properly populated.
+
+.. _Image-out-of-focus-Procedure-Overview:
+
+Overview
+========
+
+.. In one or two sentences, explain when this Image out of focus procedure needs to be used. Describe the symptoms that the user sees to use this procedure. 
+
+This procedure should be used when images appear out of focus due to a WEP failure or other related issues. Users will notice images ranging from slightly out of focus to large donut appearances when hexapod offsets are not correctly applied.
```

Comment:
" or other related issues" - like what? please specify with examples.

Reply:
It would be nice if we have an example donut images to illustrate this -- you can use the Rubin tv images from during the ops-calibs rehearsal. We had observed many out-of-focus donuts

Reply:
the image is attached 

`.. figure:: ./_static/out-focus.png
    :name: out-focus
    :width: 400
`